app/views/coronavirus_form/check_your_answers/helpers/_summary_list_multiple_items.html.erb

```diff
@@ -0,0 +1,46 @@
+<div class="gem-c-summary-list ">
+  <h2 class="govuk-heading-l gem-c-summary-list__group-title"><%= question_title %></h2>
+  <ul class="govuk-summary-list__actions-list gem-c-summary-list__group-actions-list">
+    <li class="govuk-summary-list__actions-list-item"> <a class="govuk-link" href="<%= question_href %>?change-answer">
+    Change<span class="govuk-visually-hidden"></span></a></li>
+  </ul>
+  <dl class="govuk-summary-list">
+    <% question_items.each do |item| %>
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+          <%= item[:field] %>
+        </dt>
+        <dd class="govuk-summary-list__value">
+          <% if item[:type] == :list %>
+            <%= render "coronavirus_form/check_your_answers/helpers/list", items: item[:value] %>
+          <% else  %>
+            <%= item[:value] %>
+          <% end %>
+        </dd>
+      </div>
+    <% end %>
+  </dl>
+  <h3 class="govuk-heading-m gem-c-summary-list__group-title"><%= detail_title %></h3>
+  <ul class="govuk-summary-list__actions-list gem-c-summary-list__group-actions-list">
+    <li class="govuk-summary-list__actions-list-item"> <a class="govuk-link" href="<%= detail_href %>?change-answer">
+    Change<span class="govuk-visually-hidden"></span></a></li>
```

Comment:
Here's another `span` that needs either text or to be removed.

Reply:
This is outdated 😺